models/comments.models.js

```diff
@@ -1,25 +1,26 @@
-const db = require('../db/connection')
+const db = require("../db/connection");
 // const{checkExists}=require('../utils.js')
```

Comment:
Remember to remove commented out code (looks like you have a duplicate of this line also)